src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -799,13 +800,17 @@ public partial struct UdpReceiveResult : System.IEquatable<System.Net.Sockets.Ud
         public byte[] Buffer { get { throw null; } }
         public System.Net.IPEndPoint RemoteEndPoint { get { throw null; } }
         public bool Equals(System.Net.Sockets.UdpReceiveResult other) { throw null; }
-        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhen(true)] object? obj) { throw null; }
+        public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }
         public override int GetHashCode() { throw null; }
         public static bool operator ==(System.Net.Sockets.UdpReceiveResult left, System.Net.Sockets.UdpReceiveResult right) { throw null; }
         public static bool operator !=(System.Net.Sockets.UdpReceiveResult left, System.Net.Sockets.UdpReceiveResult right) { throw null; }
     }
     public sealed partial class UnixDomainSocketEndPoint : System.Net.EndPoint
     {
         public UnixDomainSocketEndPoint(string path) { }
+        public override System.Net.Sockets.AddressFamily AddressFamily { get { throw null; } }
+        public override System.Net.EndPoint Create(System.Net.SocketAddress socketAddress) { throw null; }
+        public override System.Net.SocketAddress Serialize() { throw null; }
+        public override string ToString() { throw null; }
```

Comment:
Any idea @jkotas  why this shows now here? (or if that matters)

Reply:
It means that the implementations for these methods were inherited from the base class originally when this ref was generated and implemented later.

Adding this here is a cosmetic change. It does not really affect anything that matters.